js/source.js

```diff
@@ -20,3 +20,22 @@ var expEntries = [
     writeup:  'Marketing Coordinator developing proposals and marketing material for an engingeering and environmental sciences firm.'
   }
 ];
+
```

Comment:
Small thing, but would be good to align some of the object parameter content for readability.